server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.kt

```diff
@@ -519,6 +523,76 @@ class HumanSkeleton(
 		if (isTrackingRightArmFromController) rightHandTrackerBone.update()
 	}
 
+	private fun updateTrackerYawCorrections() {
+		val trackersToAlign = listOf(
+			headTracker,
+			upperChestTracker,
+			chestTracker,
+			waistTracker,
+			hipTracker,
+		)
```

Comment:
This is a ton of allocations (run once per update, aka ~1000 times per second). Ideally we should avoid recreating this, but at the least it should be an array. Perhaps this array should be created in `setTrackersFromList()` and cached for use here?
```suggestion
		val trackersToAlign = arrayOf(
			headTracker,
			upperChestTracker,
			chestTracker,
			waistTracker,
			hipTracker,
		)
```

Reply:
Addressed by moving it into the `init` of `SpineYawCorrection` class. I also cleaned it up by removing null and non-IMU trackers. This is updated only when the config is changed from the UI.